src/app/components/mno-app-onboarding-btn/mno-app-onboarding-btn.html

```diff
@@ -36,3 +36,10 @@
     {{ 'mno_enterprise.templates.components.app_install_btn.access_addon' | translate }}
   </a>
 </div>
+
+<div ng-show="!$ctrl.isLoadingAppInstances" ng-switch on="$ctrl.appInstance.status">
+  <div ng-switch-when="CONFLICT" class="message">
```

Comment:
This code block seems to be the same as here: https://github.com/maestrano/mno-enterprise-angular/pull/235/files#diff-1b3a0dd242b70bf1890c57746b33e1a4R6

Won't it display the error twice?